dd-sdk-android-core/api/apiSurface

```diff
@@ -265,6 +265,7 @@ interface com.datadog.android.core.internal.persistence.Deserializer<P: Any, R:
   fun deserialize(P): R?
 fun java.io.File.canReadSafe(com.datadog.android.api.InternalLogger): Boolean
 fun java.io.File.existsSafe(com.datadog.android.api.InternalLogger): Boolean
+fun java.io.File.listFilesSafe(com.datadog.android.api.InternalLogger, java.io.FilenameFilter): Array<java.io.File>?
```

Comment:
nit: arguments order should be flipped, `InternalLogger` should come last as for the other similar methods.

Reply:
if it is lambda, we can do like that indeed